### DISCUSSION TITLE
Trilium won't boot after renaming backup.db file to 'document'.

### DISCUSSION THREAD

**Discussion by Micolashh** [TriliumNext/Trilium#7324] [Q&A] [answered]
### Trilium Version

0l.63.7

### What operating system are you using?

Windows

### What is your setup?

Local (no sync)

### Operating System Version

windows

### Description

Trilium won't boot after renaming backup.db file to 'document'. I'm trying restore my backup, this has never happened before I've even tried redownloading Trilium, no luck

**Comment by Micolashh** [comment]
![bandicam 2024-08-14 14-16-31-580](https://github.com/user-attachments/assets/65821aff-7b7c-4cb7-94e3-99d5e2009bad)

The wal and shm files appear when I boot attempt to boot Trilium but it doesn't load

**Comment by eliandoran** [comment]
@Micolashh , could you try to run Trilium in a Windows Terminal to see if you can get some useful logs?
There's probably something crashing on startup.

Also, the backup was taken manually or automatically by Trilium?

**Comment by Micolashh** [comment]
> @Micolashh , could you try to run Trilium in a Windows Terminal to see if you can get some useful logs? There's probably something crashing on startup.
> 
> Also, the backup was taken manually or automatically by Trilium?

I had Trilium back up the .db files. Does it matter which I rename to document (weekly,daily,monthy)?
And how do you boot Trilium in the terminal?

**Comment by cmesoar** [comment, marked as answer]
same issue here with Linux Mint and Trillium v0.63.7 - the back up is really old (long story but short is not been using Trillium but now have to have information that was in thet backup!) 

Here is the error if trillium-safe-mode is executed: 
App db version is 228, while db version is 183. Migration needed.
ERROR: Direct migration from your current version is not supported. Please upgrade to the latest v0.60.4 first and only then to this version.

So...will install 0.60.4 and see if it'll work that way! Thanks!

OK - was able to open back up in v60!! Yeah!!
